mobile/lib/providers/backup/manual_upload.provider.dart

```diff
@@ -209,7 +207,9 @@ class ManualUploadNotifier extends StateNotifier<ManualUploadState> {
           );
         }
 
-        Set<AssetEntity> allUploadAssets = allAssetsFromDevice.nonNulls.toSet();
+        Set<BackupCandidate> allUploadAssets = allAssetsFromDevice.nonNulls
+            .map((asset) => BackupCandidate(asset: asset, albumNames: []))
```

Comment:
Does this mean that manual backups do not preserve the album info even when the setting is turned on? It would be best if uploads, from foreground / background / manual, all respect the sync settings

Reply:
Done!